public/.eslintrc.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+	"parser": "@typescript-eslint/parser",
+	"plugins": ["@typescript-eslint"],
+	"extends": [
+		"eslint:recommended",
+		"plugin:react/all"
+    ],
```

Comment:
This line seems to have spaces, while all others use tab. When I save this file, this line changes those spaces to tab. Not sure what config is doing that though.